player_alsa.h

```diff
@@ -0,0 +1,33 @@
+#include <alsa/asoundlib.h>
+
+void check(int *err, int newErr) {
```

Comment:
Oh btw, the reason why that function is written in C and not in Go is that the only way (AFAIK) to allocate the hardware parameters is to use `snd_pcm_hw_params_alloca`, which is a macro (not usable from Go) and allocates on stack (not wrappable to a C function).